src/contracts/misc/interfaces/IGhoAaveSteward.sol

```diff
@@ -90,12 +90,6 @@ interface IGhoAaveSteward {
    */
   function getGhoTimelocks() external view returns (GhoDebounce memory);
 
-  /**
-   * @notice Returns maximum value that can be assigned to GHO borrow rate.
-   * @return The maximum value that can be assigned to GHO borrow rate in ray (e.g. 0.01e27 results in 1.0%)
-   */
-  function GHO_BORROW_RATE_MAX() external view returns (uint32);
-
```

Comment:
`updateGhoBorrowRate` docs needs update too

Reply:
ty for the catch! [c636eae](https://github.com/aave/gho-core/pull/436/commits/c636eaec7a93fc35fb2961b4ee7aaee66d113797)